src/System.Private.ServiceModel/dir.props

```diff
@@ -2,7 +2,7 @@
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <Import Project="..\dir.props" />
   <PropertyGroup>
-    <AssemblyVersion>4.1.3.0</AssemblyVersion>
+    <AssemblyVersion>4.1.3.1</AssemblyVersion>
```

Comment:
@StephenBonikowsky can you please create a work item to update this version in master branch (it is 4.1.3.0 currently)?

Reply:
I'll just make the change real quick in master.